ui/app/adapters/watchable.js

```diff
@@ -60,7 +60,7 @@ export default class Watchable extends ApplicationAdapter {
       signal,
       data: params,
     }).catch(error => {
-      if (error instanceof AbortError) {
+      if (error instanceof AbortError || error.name == 'AbortError') {
```

Comment:
Hmm. These make me sad, but they're definitely not the end of the world. I think I'll open an issue tracking this particular thing. I wouldn't want to block the Ember and related deps work on it.

Reply:
ya, I don’t love it, a new issue sounds good to me so I can proceed through this slog 😆💞